addons/web/static/src/core/utils/website_scrolling.js

```diff
@@ -0,0 +1,150 @@
+// Scrolling util functions needed by the Website module and sub-modules. These
```

Comment:
Putting this in core makes it available in the backend as well. According to its name, it looks like it's only necessary for website => move it to the web/static/src/public folder? Or directly in the website/portal addons if that works (would be better actually...).

Side-note: it's a bit disappointing to have several scrolling helpers...

Reply:
> Side-note: it's a bit disappointing to have several scrolling helpers...

It sure is, that's why we only had my implementation... before someone implemented new ones, changed and moved existing ones, breaking practically all our usecases... :shrug: Thanks for the list of bugs and the work @sobo-odoo -> Maybe you can create a task with that list so we can create tests about those issues later? As we will have to refactor those utils at some point :point_down: 

> Or directly in the website/portal addons if that works

@sobo-odoo Let's move that file in `web_editor` and rename the file `scrolling.js` :+1: The web_editor app is meant to be removed in the future I guess so we'll see how we can share utils without touching `web` at that point. Maybe after `#wrapwrap` removal and other stuff, we'll get closer to be able to use the official util... :shrug: